internal/repository/prisma/event.go

```diff
@@ -50,6 +50,11 @@ func (r *eventRepository) ListEvents(tenantId string, opts *repository.ListEvent
 		TenantId: *pgTenantId,
 	}
 
+	if opts.Search != nil {
+		queryParams.Search.String = *opts.Search
```

Comment:
```suggestion
		queryParams.Search = sqlchelpers.TextFromStr(opts.Search)
```

^ can use this method which is meant to make this a little cleaner (setting `.Valid` isn't ideal) - just noticed that the `TextFromStr` method panics if the scan fails, which I'm not sure is ideal, but we also don't want to case on an error each time we do a simple string conversion, so perhaps it's alright?